Pass on HAS_SYSTEM_ASPRINTF to C. Should eventually make it upstream (and become default)
--- a/system.mk
+++ b/system.mk
@@ -176,7 +176,7 @@
 	-Wtrigraphs -Wformat -Wchar-subscripts \
 	-Wparentheses -pedantic -Wuninitialized
 
-CFLAGS=-Os $(WARN) $(DEFINES) $(INCLUDES) $(EXTRA_INCLUDES) $(shell dpkg-buildflags --get CFLAGS) $(shell dpkg-buildflags --get CPPFLAGS)
+CFLAGS=-Os $(WARN) $(DEFINES) $(INCLUDES) $(EXTRA_INCLUDES) $(shell dpkg-buildflags --get CFLAGS) $(shell dpkg-buildflags --get CPPFLAGS) -DHAS_SYSTEM_ASPRINTF=$(HAS_SYSTEM_ASPRINTF)
 LDFLAGS=-Wl,--as-needed $(LIBS) $(EXTRA_LIBS) $(shell dpkg-buildflags --get LDFLAGS)
 EXPORT_DYNAMIC=-Xlinker --export-dynamic
 
